### ISSUE TITLE
can a logistic regression/ bag-of-words-y model classify "discussion of" vs. "discussion about" qanon (the same topic)?

### ISSUE THREAD

**Issue by jwzimmer-zz** [jwzimmer/aboutvsof#4]
https://github.com/jwzimmer/aboutvsof/issues/2#issuecomment-764778415

hopefully i can use some of the text found in https://github.com/jwzimmer/aboutvsof/issues/1

**Comment by jwzimmer-zz** [comment]
Trying to find clear, similarly-formatted (as in, both articles or speeches, versus comparing tweets to articles) pro-Qanon content is kinda hard.

I might have to combine excerpts --> https://github.com/jwzimmer/aboutvsof/blob/main/proqexcerpts.txt:

==== From https://www.azmirror.com/2019/11/21/paul-gosar-conspiracy-theories-screwing-with-everyone/:

“If more people in Congress had a sense of humor, it wouldn’t be such a stale place,” he said. 

Using the Epstein meme “was merely the means to push our impeachment messaging to an entirely new audience,” Goldey said. 

“In the 24 hours after it was posted, our tweets had nearly 30 million impressions,” he added. By contrast, the two times that President Donald Trump retweeted the Arizona congressman, the tweets got about 3.5 million impressions. 

As for whether Gosar believes that Epstein didn’t commit suicide, Goldey said, “He believes when high profile people, particularly high profile pedophiles, are not held accountable, the American people have a right to ask questions. That being said, he wants to see the full investigation completed prior to drawing conclusions.”

==== From https://www.washingtonpost.com/politics/trump-praises-baseless-qanon-conspiracy-theory-says-he-appreciates-support-of-its-followers/2020/08/19/e50f8d46-e25e-11ea-8181-606e603bb1c4_story.html

President Trump gave a major boost to the baseless QAnon conspiracy theory, saying Wednesday that he appreciated the support of its followers, calling them “people that love our country.”

...

“I don’t know much about the movement; I understand they like me very much, which I appreciate,” Trump said during a White House press briefing. “I heard these are people that love our country.”

“QAnon believes you are secretly saving the world from this cult of pedophiles and cannibals. Are you behind that?” a reporter pressed.

“I haven’t heard that. Is that supposed to be a bad thing or a good thing?” Trump responded. “If I can help save the world from problems, I am willing to do it. I’m willing to put myself out there. And we are actually, we’re saving the world.”

==== From https://nymag.com/intelligencer/2019/07/does-the-jeffrey-epstein-indictment-qanon.html: this whole article is about as close to endorsing Qanon as I've been able to find?

==== From https://www.theatlantic.com/magazine/archive/2020/06/qanon-nothing-can-stop-what-is-coming/610567/

If you were an adherent, no one would be able to tell. You would look like any other American. You could be a mother, picking leftovers off your toddler’s plate. You could be the young man in headphones across the street. You could be a bookkeeper, a dentist, a grandmother icing cupcakes in her kitchen. You may well have an affiliation with an evangelical church. But you are hard to identify just from the way you look—which is good, because someday soon dark forces may try to track you down. You understand this sounds crazy, but you don’t care. You know that a small group of manipulators, operating in the shadows, pull the planet’s strings. You know that they are powerful enough to abuse children without fear of retribution. You know that the mainstream media are their handmaidens, in partnership with Hillary Clinton and the secretive denizens of the deep state. You know that only Donald Trump stands between you and a damned and ravaged world. You see plague and pestilence sweeping the planet, and understand that they are part of the plan. You know that a clash between good and evil cannot be avoided, and you yearn for the Great Awakening that is coming. And so you must be on guard at all times. You must shield your ears from the scorn of the ignorant. You must find those who are like you. And you must be prepared to fight.

You know all this because you believe in Q.

==== From https://www.rev.com/blog/transcripts/donald-trump-laura-ingraham-interview-transcript-august-31-says-people-in-the-dark-shadows-controlling-biden --> a fox news interview with trump, so relatively pro-conspiracy

==== From https://www.small-screen.co.uk/out-of-shadow-youtube-documentary-jeffrey-epstein-disney/ --> fairly pro-conspiracy

==== From https://theminaretonline.com/2020/05/02/an-overview-of-the-out-of-shadows-documentary/ --> relatively friendly to the conspiracy?

==== From https://www.outofshadows.org/ (note: they have a newsletter I could subscribe to but I'm kinda scared)

The Out Of Shadows documentary lifts the mask on how the mainstream media & Hollywood manipulate & control the masses by spreading propaganda throughout their content. Our goal is to wake up the general public by shedding light on how we all have been lied to & brainwashed by a hidden enemy with a sinister agenda.

 

This project is the result of two years of blood, sweat and tears by a team of woke professionals. It’s been independently produced and funded and is available on many different platforms for FREE for anyone to watch. Patriots made this documentary with the sole purpose of getting the truth out there. If you like the documentary, please share this video.

​

You can support our team and future projects by making a donation below.

**Comment by jwzimmer-zz** [comment]
A pro-Q site! Lots of their content is social media posts and videos... it's hard to find TEXT for it...

https://timothycharlesholmseth.com/child-rescue-operation-unfolding-right-now-in-nyc/
https://www.writeintoaction.com/

At least Q-adjacent: https://www.transformationtalkradio.com/archives

Has some quotes from MTG: https://www.politico.com/news/2021/01/13/crow-condemns-taylor-greene-rhetoric-458705

**Comment by jwzimmer-zz** [comment]
Liz Crokin is a q supporter, there are some things she has written: https://observer.com/author/liz-crokin/

**Comment by jwzimmer-zz** [comment]
Doing no text cleaning, just reading in the text from every p on each website page, and labeling them 1/0 according to which article they were in. Using the examples of of-q and about-q content I found here (not ideal). So trying to stick as close to within the same topic (qanon theories) as possible, a b-o-w model does an ok job on distinguishing the types of speech. (https://github.com/jwzimmer/aboutvsof/commit/e67b5b37459e172a4232e976cf57698656332912)

Training Accuracy:  0.9989785495403473
Validation Accuracy:  0.8653061224489796

![image](https://user-images.githubusercontent.com/1320456/105412113-45764180-5c02-11eb-8fc2-683065f49909.png)

Here's how I categorized the files... articles about QAnon related topics or people are categorized as 0, articles relatively sympathetic to Q are labeled 1, articles written by Liz Crokin are labeled 1 (since she is a Q supporter), some excerpted quotes I found were saved and labeled as 1 but I don't think I ended up using that file (the .txt file):
```
      "‘Depraved’_ Rep. Jason Crow condemns Rep. Marjorie Taylor Greene’s rhetoric ahead of impeachment vote - POLITICO.html":0,
        "‘He knows he’s screwing with everyone’_ Gosar embraces conspiracy theories.htm":0,
        "An overview of the “Out of Shadows” documentary – Est. 1933.html":1,
        "Apple, Twitter, Google, and Instagram Collude to Defeat Trump _ Observer.htm":1,
        "Creative Coalition Rejects $10k Donation From Veterans—Because ‘Trump’ _ Observer.htm":1,
        "Does the Jeffrey Epstein indictment prove Qanon right_.htm":1,
        "Donald Trump Laura Ingraham Interview Transcript August 31_ Says People _in the Dark Shadows_ Controlling Biden - Rev.htm":1,
        "Exclusive_ Hillary Clinton Campaign Systematically Overcharging Poorest Donors _ _ Observer.htm":1,
        "Facebook and YouTube Show Frantic Alliegence to Clinton _ Observer.htm":1,
        "Fact check_ 35,000 “malnourished” and “caged” children were not recently rescued from tunnels by U.S. military _ Reuters.html":0,
        "House votes to condemn baseless QAnon conspiracy theory - The Washington Post.htm":0,
        "How QAnon Became Obsessed With ‘Adrenochrome,’ an Imaginary Drug Hollywood Is Harvesting from Kids.htm":0,
        "Inside ‘Out of Shadows,’ the Bonkers Hollywood-Pedophilia Documentary QAnon Loves.htm":0,
        "Is YouTube Preventing People From Finding The Out Of Shadows Documentary_ _ Small Screen.htm":1,
        "Media Favoritism of Clinton Extends Beyond News Programs _ _ Observer.htm":1,
        "Media Orgs Donate to Clinton Foundation Then Downplay Clinton Foundation Scandal _ _ Observer.htm":1,
        "Milo Yiannopoulos Advises Donald Trump to Quit Twitter _ Observer.htm":1,
        "National Enquirer Never Would Have Run BuzzFeed’s Trump Dossier _ Observer.htm":1,
        "proqexcerpts.txt":1,
        "QAnon and Child Trafficking_ Is 'Save the Children' a Conspiracy_.htm":0,
        "QAnon Is More Important Than You Think - The Atlantic.htm":0,
        "QAnon Promotes Pedo-Ring Conspiracy Theories. Now They’re Stealing Kids..html":0,
        "QAnon_ the religion of conspiracy - Elcano Blog.htm":0,
        "Shia LaBeouf’s Behavior at Trump Protest Is Hypocrisy at Its Finest _ Observer.htm":1,
        "Team Trump Isn’t Hiding Support for QAnon Kooks Anymore After Marjorie Taylor Greene’s Win in Georgia Primary.htm":0,
        "The Dark Virality of a Hollywood Blood-Harvesting Conspiracy _ WIRED.htm":0,
        "Timothy Holmseth - Guest Profile.htm":1,
        "Timothy Holmseth Update_ Deep State Harassment, Jury Trial _ Listen Notes.htm":1,
        "Viral Chart Distorts Human Trafficking Statistics - FactCheck.org.html":0
```

**Comment by jwzimmer-zz** [comment]
Not exactly qanon, but voter fraud of vs. about: https://voterfraud2020.io/, from @janeadams

**Comment by jwzimmer-zz** [comment]
Some pro-q articles! https://www.martingeddes.com/the-digital-coup-and-the-great-exposure/

More pro-q articles: https://www.stitcher.com/show/those-conspiracy-guys/episode/praying-medic-qanon-68988537, https://prayingmedic.com/blog/

**Comment by jwzimmer-zz** [comment]
document embedding word embedding, semantic space, bert, word2vec, glove embeddings
https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html#examples-using-sklearn-feature-extraction-text-tfidfvectorizer

**Comment by jwzimmer-zz** [comment]
I think reasonable proof of concept established... for now moving on to tweets
